tools/shell/test/test_shell_control_edit.py

```diff
@@ -61,7 +60,8 @@ def test_enter(temp_db, key, history_path):
     f = open(os.path.join(history_path, "history.txt"))
     assert f.readline() == 'RETURN "databases rule" AS a;\n'
     f.close()
-    shutil.rmtree(os.path.join(history_path, "history.txt"), ignore_errors=True)
+    if os.path.exists(os.path.join(history_path, "history.txt")):
```

Comment:
I think you can add a test utils file, and refactor this as a function: deleteIfExists:
def deleteIfExists(file):
   if os.path.exists(file):
       os.remove(file)